docs/camino-node/azure.md

```diff
@@ -0,0 +1,278 @@
+---
+sidebar_position: 12
+title: Run Camino Node with Microsoft Azure
+description: This guide provides step-by-step instructions on how to set up and run Camino Node on the Microsoft Azure.
+---
+
+# Setting Up Camino Node on Azure
+
+In this guide, we will walk you through the process of setting up Camino Node on the Microsoft Azure platform.
+You have two methods to choose from for the setup process: the manual method and the automatic method using Terraform.
+
+Camino Node is a decentralized messaging and booking platform built on blockchain technology, offering secure
+and efficient communication and transaction capabilities within the Camino Network. By following this guide,
+you'll be able to deploy Camino Node on Azure and explore the potential of decentralized messaging in the travel industry.
+
+Whether you're new to blockchain technology or an experienced Azure user, this guide is designed to be accessible
+and easy to follow. It provides step-by-step instructions to help you get Camino Node up and running smoothly on Azure.
+
+Let's get started with the setup process of Camino Node on Azure. Choose the method that best suits your preferences and needs.
+
+## Automatic Method
+
+Before you begin the setup process on Microsoft Azure, ensure you have the `template.json` and `parameters.json` files. You can download them using the links below:
+
+- [`template.json`](/img/azure/template.json)
+- [`parameters.json`](/img/azure/parameters.json)
+
+:::caution Rename Files After Download
+
+The downloaded files have UUIDs appended to their names. After downloading, please rename them
+to `template.json` and `parameters.json` respectively.
+
+:::
+
+### 1: Editing the `parameters.json` File
+
+Before initiating the setup, you'll need to make a few changes to the `parameters.json` file:
+
+1. Modify lines 40 & 55 to input your IP address.
+2. On line 172, input your public SSH key. If you don't have an SSH key ready, follow the instructions below.
+
+### 2: Generate an SSH Key
+
+Execute the following commands to create an SSH key:
+
+```bash
+ssh-keygen -m PEM -t rsa -b 4096 -f $(pwd)//azurekey -N ""
+```
+
+Next, open `azurekey.pub` and copy its contents. This will be pasted into line 172 of the `parameters.json` file.
+
+### 3: Install Azure CLI
+
+Install the Azure Command-Line Interface (CLI) to interact with the Azure resources. Find detailed installation instructions [here](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+For Ubuntu systems, you can directly install using:
+
+```bash
+curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+```
+
```

Comment:
Sorry I forgot to add a step here to login 

```bash
az login
```

Reply:
Fixed